### PR TITLE
Removing pydantic check for price_level_schedule and absolute_price_schedule

### PR DESCRIPTION
**Pull request by SebaLukas**
## Describe your changes
See title

## Issue ticket number and link
According the iso15118-20 schema file the charger can either send no price information, absolute or price_level

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [x] I read the [contribution documentation](https://github.com/EVerest/EVerest/blob/main/CONTRIBUTING.md) and made sure that my changes meet its requirements

### CODE REVIEW

**Review by a-w50 — COMMENTED** [review]
Instead of deleting the check, wouldn't it be possible to extend it, so that it also gets validated if none of the properties is set?

**Comment by SebaLukas** [comment]
@a-w50 I have added the wanted check

File: iso15118/shared/validators.py [hunk 1]
```diff
@@ -78,3 +78,36 @@ def one_field_must_be_set(
...
+
+    Args:
+        field_options: List of optional field names and aliases of a model.
+                       For each field, we need both the field name and the alias
+                       because when instantiating a pydantic model, we use the
+                       pythonic field names, but when de-serialising the model
+                       through JSON via the EXI codec, we use the aliases.
+        values: The dict with the model's fields
+    """
+    set_fields: List = []
```
Comment (a-w50) [inline iso15118/shared/validators.py]: these lines could probably written as:
```
count = sum(values.get(field_name) is not None for field_name in field_options);
```
or
```
field_values = [values.get(field_name) for field_name in field_options]
set_fields = [x for x in field_values if x is not None]
```

**Review by a-w50 — APPROVED** [review]
:+1:

**Pull request merged**